spec/v2/index.bs

```diff
@@ -1602,23 +1621,40 @@ content-length: <l>[=ContentLength=]</l>
 <l>[=AuthBody=]</l>
 </pre>
 
+In addition, if a host system supports OpenId Connect, the host system CAN offer and implement this Action under a second URL, and set this URL as the value of `token_endpoint` of the [=OpenId Provider Configuration Document=].
+
+<pre highlight=http>
+POST <l>[=AuthEndpoint=]</l> HTTP/1.1
+accept: application/json
+content-type: application/x-www-form-urlencoded
+authorization: Basic <l>[=BasicAuth=]</l>
+content-length: <l>[=ContentLength=]</l>
+
+<l>[=AuthBody=]</l>
+</pre>
+
 With Request parameters:
 
-: <dfn>AuthSubpath</dfn>: 
-:: If a [=host system=] uses a relative subpath dedicated to the purpose of creating an [=access token=], then the requesting [=data recipient=] MUST prepend this subpath.
+: <dfn>AuthEndpoint</dfn>
+:: The endpoint to request an [=access token=] after discovering the value by performing the [=authentication flow=].
+
+: <dfn>AuthSubpath</dfn>:
+:: If a [=host system=] uses a relative subpath dedicated to serving an [=OpenId Provider Configuration Document=] and creating an [=access token=], then the requesting [=data recipient=] MUST use this subpath.
 
 : <dfn>AuthHostname</dfn>
-:: The requesting data recipient MUST use the domain name of the host system dedicated for the purpose of creating an authentication token.
+:: The requesting data recipient MUST use the domain name of the host
+    system dedicated to serving an [=OpenId Provider Configuration Document=] and creating an [=access token=].
 
 : <dfn>BasicAuth</dfn>
 :: See [[!rfc6749]] Section 4.4.2
 
 : <dfn>AuthBody</dfn>
-:: See [[!rfc6749]] Section 4.4
+:: See [[!rfc6749]] Section 4.4. If successful, contains the <dfn>access token</dfn>.
```

Comment:
Does this addition belong here? As the access_token (in case of success) is part of the response (hence, irrelevant for request syntax).